Tests/test_metaclass.py

```diff
@@ -380,38 +381,37 @@ class Foo(test=1, **{'metaclass':funcMeta, 'test2':2}): pass
 
     def test_keyword_arguments_class(self):
         # type() does not accept any keyword arguments
-        with self.assertRaisesRegex(TypeError, r"^type\(\) takes .* arguments"):
+        with self.assertRaisesRegex(TypeError, r"\(\) takes .* arguments"):
             class Foo(test=True): pass
 
-        with self.assertRaisesRegex(TypeError, r"^type\(\) takes .* arguments"):
+        with self.assertRaisesRegex(TypeError, r"\(\) takes .* arguments"):
             class Foo(test=True, **{}): pass
 
-        with self.assertRaisesRegex(TypeError, r"^type\(\) takes .* arguments"):
+        with self.assertRaisesRegex(TypeError, r"\(\) takes .* arguments"):
             class Foo(test=True, metaclass=type): pass
 
         class MetaP(type):
             @classmethod
             def __prepare__(metacls, name, bases, **kwargs):
                 return type.__prepare__(metacls, name, bases)
 
-        msg = "MetaP() takes at most 4 arguments (5 given)" if is_cli else "type() takes 1 or 3 arguments"
-        with self.assertRaisesMessage(TypeError, msg):
+        with self.assertRaisesRegex(TypeError, r"\(\) takes .* arguments"):
             class Foo(test=True, metaclass=MetaP): pass
 
-        class MetaN(type):
-            def __new__(metacls, name, bases, attrdict, **kwargs):
-                return type.__new__(metacls, name, bases, attrdict)
+        if (sys.version_info < (3,6)):
```

Comment:
What's the behavior for >= (3,6), is it something we could easily adopt? Should we have a negative test to make sure we fail when we get to 3.6?